### PR TITLE
update for ocp-13032 failed

### PR DESCRIPTION
**Pull request by zhouying7780**
@kasturinarra  could you help review ? thanks.

### CODE REVIEW

File: features/upgrade/workloads/oc-upgrade.feature [hunk 1]
```diff
@@ -18,29 +18,28 @@ Feature: basic verification for upgrade oc client testing
...
     Given status becomes :running of 1 pods labeled:
-      | name=httpd-example |
+      | deploymentconfig=hello-openshift |
     When I run the :rsh client command with:
-      | pod     | <%= pod.name %>            |
-      | command | ls                         |
-      | command | /etc                       |
+      | pod     | <%= pod.name %>          |
+      | command | ls                       |
+      | command | /etc                     |
```
Comment (liangxia) [inline features/upgrade/workloads/oc-upgrade.feature]: Extra white-spaces before the last `|`

File: features/upgrade/workloads/oc-upgrade.feature [hunk 2]
```diff
@@ -7,7 +7,7 @@ Feature: basic verification for upgrade oc client testing
     When I run the :new_project client command with:
       | project_name | workloads-upgrade |
     When I run the :new_app client command with:
-      | template | httpd-example |
+      | docker_image   | aosqe/hello-openshift |
```
Comment (liangxia) [inline features/upgrade/workloads/oc-upgrade.feature]: Extra white-spaces before the seconds `|`

File: features/upgrade/workloads/oc-upgrade.feature (hunk 1 above)
Comment (zhouying7780) [reply features/upgrade/workloads/oc-upgrade.feature]: thanks, have updated.

**Comment by zhouying7780** [comment]
@lxia  Have updated , please help review again. thanks.

**Pull request merged**